website/docs/r/cloud_project_region_storage_presign.html.markdown

```diff
@@ -29,14 +27,16 @@ output "presigned_url" {
 
 The following arguments are supported:
 
-* `service_name` - (Required) The id of the public cloud project. If omitted,
+- `service_name` - (Required) The id of the public cloud project. If omitted,
   the `OVH_CLOUD_PROJECT_SERVICE` environment variable is used.
-* `region_name` - (Required) The region in which your storage is located.
-  Ex.: "GRA".
-* `name` - (Required) The name of your S3 storage container/bucket.
-* `expire` - (Required) Define, in seconds, for how long your URL will be valid.
-* `method` - (Required) The method you want to use to interact with your object. Can be either 'GET' or 'PUT'.
-* `object` - (Required) The name of the object in your S3 bucket.
+- `region_name` - (Required) The region in which your storage is located. Must
+  be in **uppercase**. Ex.: "GRA".
```

Comment:
If the region must be in uppercase, should the terraform provider take care of putting it in uppercase if it was supplied in lower case ?

for example here: https://github.com/ovh/terraform-provider-ovh/blob/5bbcd922d8601765baad74dc83a38e411e63cb4b/ovh/resource_cloud_project_region_storage_presign.go#L76

This would prevent confusion when some documentation state that regions must be in lower case: https://help.ovhcloud.com/csm?id=kb_article_view&sysparm_article=KB0047384
![image](https://github.com/user-attachments/assets/7df2e2dc-dc62-4f83-9390-9f25989fd754)